plugin/wavesurfer.minimap.js

```diff
@@ -94,7 +94,7 @@ WaveSurfer.Minimap = WaveSurfer.util.extend({}, WaveSurfer.Drawer, WaveSurfer.Dr
 
     bindWaveSurferEvents: function () {
         var my = this;
-        this.wavesurfer.on('ready', this.render.bind(this));
+        this.wavesurfer.on('ready', my.render());
```

Comment:
`this.wavesurfer.on('ready', my.render());` is certainly wrong, since it's calling the render function immediately.

You need to either subscribe like the original code:
```
this.wavesurfer.on('ready', this.render.bind(this))):
``` 

Or simply call `this.render()`.